ckanext/xloader/plugin.py

```diff
@@ -212,3 +229,40 @@ def get_helpers(self):
             "xloader_status": xloader_helpers.xloader_status,
             "xloader_status_description": xloader_helpers.xloader_status_description,
         }
+
+
+def _should_remove_unsupported_resource_from_datastore(res_dict_or_obj):
+    if not toolkit.asbool(toolkit.config.get('ckanext.xloader.clean_datastore_tables', False)):
+        return False
+    if isinstance(res_dict_or_obj, Resource):
+        res_dict_or_obj = res_dict_or_obj.__dict__
+    return ((not XLoaderFormats.is_it_an_xloader_format(res_dict_or_obj.get('format', u''))
+                or res_dict_or_obj.get('url_changed', False))
```

Comment:
What is the relevance of `url_changed` here?

Reply:
@ThrawnCA same thing as other PR, just a specific thing in our setup to my new understanding. As generally, systems can fetch from external source. E.g. a CSV file hosted somewhere else can still be fetched by Xloader and loaded into the datastore?

Reply:
An external CSV can be loaded, yes, but then the format field should be CSV.